modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -125,11 +125,9 @@ public ObjectEntry getByExternalReferenceCode(String externalReferenceCode)
 
 	@Override
 	public EntityModel getEntityModel(MultivaluedMap multivaluedMap) {
-		if (_entityModel == null) {
-			_entityModel = new ObjectEntryEntityModel(
-				_objectFieldLocalService.getObjectFields(
-					_objectDefinition.getObjectDefinitionId()));
-		}
+		_entityModel = new ObjectEntryEntityModel(
+			_objectFieldLocalService.getObjectFields(
+				_objectDefinition.getObjectDefinitionId()));
```

Comment:
@p-albuquerque can we return right away? Thx

Reply:
Okay.